product_yahoo_auction_sst/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Quartile Limited
```

Comment:
```suggestion
# Copyright 2017-2018 Quartile Limited
```

Do not remove the years of copyright.  If there are some material changes, then we update the end year to the current year.  No need to change if there is no material change.  Please apply this to the other files as well.